lib/sprinkle/installers/npm.rb

```diff
@@ -16,26 +16,26 @@ class Npm < Installer
       attr_accessor :package_name
 
       api do
-        def npm(package, &block)
-          install Npm.new(self, package, &block)
+        def npm(package, options={}, &block)
+          install Npm.new(self, package, options, &block)
         end
       end
 
       verify_api do
         def has_npm(package)
-          @commands << "npm --global list | grep \"#{package}@\""
+          @commands << "npm --global list | grep -F \"#{package}@\""
```

Comment:
I think to be correct her we need to match " [package]" (space in front), based on the output - to avoid false matches, yes?  I'd say fix this and we can merge.